pkg/flare/archive_win.go

```diff
@@ -180,79 +182,60 @@ func exportWindowsEventLog(fb flaretypes.FlareBuilder, eventLogChannel, eventLog
 
 func getServiceStatus(fb flaretypes.FlareBuilder) error {
 	return fb.AddFileFromFunc(
-		"servicestatus.txt",
+		"servicestatus.json",
 		func() ([]byte, error) {
-			cancelctx, cancelfunc := context.WithTimeout(context.Background(), execTimeout)
-			defer cancelfunc()
-
-			cmd := exec.CommandContext(cancelctx, "powershell", "-c", "get-service", "data*,ddnpm", "|", "fl")
-
-			var out bytes.Buffer
-			cmd.Stdout = &out
-			err := cmd.Run()
+			manager, err := winutil.OpenSCManager(scManagerAccess)
```

Comment:
I feel that a bulk of this function should be moved into new module already dedicated to both flare and services

Reply:
Hey @iglendd , could you elaborate on moving the function to a separate module. Do you mean possibly moving most of the function into another function in `service_windows.go`?

Reply:
Yes, @clarkb7 WDYT?

Reply:
right now I don't see a benefit to having it in one place over another. But with these things it's usually faster to move it than it is to spend time trying to justify it one way or the other.

Reply:
I created a new function `getDDServices` that does the service information collection after connecting to the SC Manager.